### PR TITLE
Feature/gallery container style

### PR DESCRIPTION
**Pull request by venegasgabby76**
### Pull Request Template

**Description**

- adjusted the media queries to be more responsive

**motivation/purpose for the feature**

- make sure that the gallery is responsive.

**What were those changes**

- made appropriate changes to the galleryview.less file

Commits Checklist
[X ] commits have clear title names
[ X] commit has descriptive message of what has changed
[X ] more small commits than less big commits

**Loom Video**

- https://www.loom.com/share/d26810e79a9d4a82af0bba4346f8b2b6

PRs needs to be reviewed by at least 2 team members
TPL is the 3rd reviewer
Lots of comments and suggestions, please!
Pull Request Rubric

### CODE REVIEW

File: src/components/pages/Gallery/Weekly.js [hunk 1]
```diff
@@ -39,6 +40,9 @@ const Weekly = props => {
             <div className="sub-container">
               <img
                 className="gallery-submission"
+                // {...pageUrl.map(page => (
```
Comment (chukalicious) [inline src/components/pages/Gallery/Weekly.js]: If you're not going to use this block it'd be good to delete it

**Review by chukalicious — DISMISSED** [review]
Thank you, Gaby! More changes to the data are expected this week, which means the props in these components will need to be further updated but this is a good start!

File: src/components/pages/Gallery/Weekly.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ const Weekly = props => {
     setIsModalVisible(false);
   };
 
+  console.log(pageUrl);
```
Comment (milklot) [inline src/components/pages/Gallery/Weekly.js]: I will advise comment or delete the console.log after use when you don't need it

**Review by milklot — DISMISSED** [review]
Good job on a responsive styling , I will advise comment or delete the console.log after use when you don't need it

**Review by ashtilawat — CHANGES_REQUESTED** [review]
Let's remove that console.log then and I will merge it ASAP.

**Review by ashtilawat — APPROVED** [review]

**Review by ashtilawat — APPROVED** [review]

**Review by chukalicious — APPROVED** [review]

**Review by Aaron-TheCreator — APPROVED** [review]
We love some good responsive design. Thank you for having clean code, good less fundamentals.

**Pull request merged**